post/views.py

```diff
@@ -0,0 +1,50 @@
+'''
```

Comment:
Will clean up. The comments were there more as a reference to myself, just in case. Just a first round. I need to do some refactoring before it becomes final.